.github/workflows/dependency-updater.yml

```diff
@@ -0,0 +1,58 @@
+name: dependency-updater
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
```

Comment:
What happens if a pr is open with a dependency bump, but it's just not merged in yet?

Reply:
Nothing, the PR will be left open and the changes untouched (the step `Create Pull Request` will try to update the branch, see no changes and does nothing to the PR)

Reply:
Cool, thanks for clarification!